docs/reference/commands/init.rst

```diff
@@ -0,0 +1,52 @@
+
+.. _ref_commands_init:
+
+init
+====
+
+Initialise a project.
+
+If '<project-dir>' is provided, initialise in that directory,
+otherwise initialise in the current working directory.
+
+If '--name <name>' is provided, the project will be named '<name>'.
+Otherwise, the project will be named after the directory it is initialised in.
+
+'--profile <profile>' is used to initialise the project for a specific use case.
+
+Init can work in an existing project directory. If there are any files in the
+directory that would be overwritten, then init command will fail.
```

Comment:
What do you think about my reviewing the command strings as a 25.10 task, for alignment with our CLI specs? I've done it for some of the other new apps.

Reply:
This piece of doc is automatically generated from the CLI and in this case is not specific to imagecraft.
So I guess any CLI rework will happen in craft-application. Unless you meant something else?

Reply:
That's what I meant. :) It's older content that preades both of us.

The reason I suggested it is because we're bubbling up content that normally doesn't get seen outside of the CLI, so we can double our value if I review it once and improve it in both docs and CLI.